test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_handlers_test.clj

```diff
@@ -171,6 +173,41 @@
           (is (= (:status response) 200))
           (is (= (:body response) init-param-two)))))))
 
+(deftest websocket-test
+  (testing "Websocket handlers"
+    (with-app-with-config app
+      [jetty9-service]
+      jetty-plaintext-config
+      (let [s                     (get-service app :WebserverService)
+            add-websocket-handler (partial add-websocket-handler s)
+            path                  "/test"
+            connected             (atom 0)
+            server-messages       (atom [])
+            client-messages       (atom [])
+            handlers              {:on-connect (fn [ws]
+                                                 (ws-session/send! ws "Hello client!")
+                                                 (swap! connected inc))
+                                   :on-text    (fn [ws text]
+                                                 (ws-session/send! ws (str "You said: " text))
+                                                 (swap! server-messages conj text))}]
```

Comment:
@cprice404 @richardc I agree, even if this feature is experimental I think it would be best to have a basic test for all the handler types to ensure everything is working properly.